c/writebuf.cc

```diff
@@ -76,25 +76,43 @@ FileWritableBuffer::~FileWritableBuffer() {
 }
 
 
-size_t FileWritableBuffer::prep_write(size_t size, const void* src) {
+size_t FileWritableBuffer::prep_write(size_t src_size, const void* src)
```

Comment:
Any way to parallelize this function? Or it is fast enough?

Reply:
Yes, it's implemented in class `MmapWritableBuffer`. However, it does not always work inside docker instances.